.gitignore

```diff
@@ -1,2 +1,4 @@
 target
 Cargo.lock
+
+.idea/
```

Comment:
Hmm, let's not put non rusty stuff in our .gitignore files.

Reply:
This is actually from Intellij Rust
but I understand. I will remove.

Reply:
done